File: single-language-analysis/dataset-files/java/TestSmell066.java
// Original URL: https://github.com/elaissoussi/exercices/blob/50ac9b6d9e89c094010a9ed637bbd3198f36a729/marsrover/MarsRoverTest.java#L38-L48

public class TestSmell066 {
	@Test
	@Ignore
	public void driveEstThanTurnLeft() {
		String expectedPath = new StringBuilder().append("    *")
				.append(LINE_SEPARATOR).append("    |").append(LINE_SEPARATOR)
				.append("    |").append(LINE_SEPARATOR).append("    |")
				.append(LINE_SEPARATOR).append("X---+").append(LINE_SEPARATOR)
				.toString();
		System.out.println(expectedPath);
		assertThat(new MarsRover("sssslssss").path(), equalTo(expectedPath));
	}
}

File: single-language-analysis/dataset-files/python/test_smell_066.py
import pytest

class TestMarsRover:
    @pytest.mark.skip
    def test_smell_066(self):
        expected_path = "    *\n" \
                        "    |\n" \
                        "    |\n" \
                        "    |\n" \
                        "X---+\n"
        print(expected_path)
        assert MarsRover("sssslssss").path() == expected_path